src/services/transactions/index.ts

```diff
@@ -18,13 +18,15 @@ export const getTxDetails = memoize(
   (id: string, chainId: string) => `${chainId}-${id}`,
 )
 
-export const getTxHistory = (chainId: string, safeAddress: string, trusted = false, pageUrl?: string) => {
+export const getTxHistory = (chainId: string, safeAddress: string, hideSuspicious = false, pageUrl?: string) => {
```

Comment:
We should maybe have two flags for trusted and imitation that we pass here.

The trusted filtering requires token lists to be available. But imitations don't.

Reply:
Created separate flags here and also in fetchFilteredTxHistory